scarb-metadata/Cargo.toml

```diff
@@ -23,6 +23,7 @@ thiserror = { workspace = true, optional = true }
 
 [dev-dependencies]
 assert_fs.workspace = true
+cairo-lang-filesystem.workspace = true
```

Comment:
Is `dev-dependencies` enough? We use it in `scarb/src/ops/resolve.rs` as well https://github.com/software-mansion/scarb/blob/0a5f7e5e539ce010bd9de0dd7b6a0ae0f7d6182b/scarb/src/ops/resolve.rs#L4